lib/client/signalflow/request_manager.js

```diff
@@ -154,6 +154,10 @@ function RequestManager(endPoint) {
       activeSocket = new WebSocket(wsEndpoint + '/v2/signalflow/connect');
       activeSocket.binaryType = 'arraybuffer';
       activeSocket.onmessage = onWebSocketMessage;
+      activeSocket.onerror = function (evt) {
+        console.error('Socket error: ' + evt);
```

Comment:
This line looks redundant. It will print the error on console by default when it's thrown.

Reply:
removed it